vertica_python/vertica/connection.py

```diff
@@ -204,10 +348,10 @@ def __init__(self, options=None):
         self.options.setdefault('unicode_error', None)
         self._cursor = Cursor(self, self._logger, cursor_type=None,
                               unicode_error=self.options['unicode_error'])
-
-        self._logger.info('Connecting as user "{}" to database "{}" on host "{}" with port {}'.format(
-                     self.options['user'], self.options['database'],
-                     self.options['host'], self.options['port']))
+        if self._logger.isEnabledFor(logging.INFO):
```

Comment:
According to https://docs.python.org/3.6/howto/logging.html#optimization, I don't think it is worthwhile to use ```isEnabledFor()```. The arguments passed to the logging method are dict values, they are not expensive. I worry that ```isEnabledFor()``` can itself be more expensive.